server/agoraApi.yaml

```
@@ -144,7 +144,7 @@ paths:
     delete:
       tags: 
         - Tags
-      summary: Delete an existing tag providing it's Id
+      summary: Delete an existing tag provided it's Id
```

Loving the clean-up effort of the existing code. Additionally suggesting the following for conciseness and consistency with the above edit:

```suggestion
      summary: Deletes existing tag by Id
```